File: Desenvolvimento/server/src/ApplicationCore/Entities/FluxoAcao.cs
using ApplicationCore.Interfaces.Base;
using System;
using System.Collections.Generic;
using System.Text;

namespace ApplicationCore.Entities
{
    public class FluxoAcao : IBaseEntity<long>
    {
        public long Id { get; set; }
        public long FluxoId { get; set; }
        public string Nome { get; set; }
        public virtual Fluxo Fluxo { get; set; }
    }
}

File: Desenvolvimento/client/src/@fuse/types/models/fluxo-acao.ts
import { Fluxo } from './fluxo';
import { BaseModel } from '../base-model';

export class FluxoAcao extends BaseModel {
    constructor(
        public id: number,
        public nome?: string,
        public fluxoId?: number,
        public fluxo?: Fluxo
    ) { super(); }
}
